### ISSUE TITLE
Make Acid simpler to build?

### ISSUE THREAD

**Issue by spiralsam** [EQMG/Acid#101]
I notice there are a number of dependencies to get Acid to complete a first-time build. For someone who first wants to get an idea of the project and not necessarily want to commit to toying with it yet, it's a bit daunting.

Luckily many are available in vcpkg, though some aren't (particularly the third_party). Perhaps to make trying out Acid, the project could do either:

1) provide prebuilt releases,
2) add dependencies as submodules that auto-build as part of the main build,
3) ensure properly versioned dependencies are available as part of vcpkg, including Acid itself

**Comment by mattparks** [comment]
I agree that I should set up a prebuilt release pipeline. The libraries I included into `third_party` consists of small libraries that are are built into Acid, they are not found on the system. `Sources/CMakeLists.txt` is where the dependency management code lives, if a dependency is not installed on the system Acid will use `FetchContent` to build the dependency in-source. I've kept the package min-version dependency requirements in Acid at or below the current vcpkg port versions.

For building on Windows you only really need the OpenAL SDK, and Vulkan SDK installed. I have vcpkg listed as optional because having some dependencies pre-built accelerates development.
